### PR TITLE
DCD-652: Creates a cloudwatch dashboard conditionally, if cloudwatch …

### PR DESCRIPTION
**Pull request by theghostwhoforks**
…integration is enabled

### CODE REVIEW

File: templates/quickstart-confluence-master.template.yaml [hunk 1]
```diff
@@ -1721,6 +1721,21 @@ Resources:
       FromPort: -1
       ToPort: -1
       SourceSecurityGroupId: !Ref SecurityGroup
+# Optional: Cloudwatch dashboard to be created when CloudWatch is enabled
```
Comment (nanux) [inline templates/quickstart-confluence-master.template.yaml]: `CloudWatch`

File: templates/quickstart-confluence-master.template.yaml [hunk 2]
```diff
@@ -1721,6 +1721,21 @@ Resources:
...
+  CloudWatchDashboard:
+    DependsOn:
+      - DB
```
Comment (nanux) [inline templates/quickstart-confluence-master.template.yaml:1727]: What would happen if the `DB` wasn't in the `DependsOn`? Will it create the dashboard anyway or will it fail?

**Review by nanux — APPROVED** [review]

File: templates/quickstart-confluence-master.template.yaml (hunk 2 above)
Comment (theghostwhoforks) [reply templates/quickstart-confluence-master.template.yaml:1727]: I'm not completely sure - I reckon CFn will wait for the DB anyway as the resource is referred to. Will check and update

Comment (theghostwhoforks) [reply templates/quickstart-confluence-master.template.yaml:1727]: As expected, CFn is waiting for provisioning completion before going ahead to provision the dashboard

**Pull request merged**